catalog/core/catalog-core-api/src/main/java/ddf/catalog/source/IngestException.java

```diff
@@ -16,6 +16,7 @@
 /**
  * The Class {@link IngestException} is thrown by a {@link CatalogProvider} when an error occurs on
  * an operation.
+ * 
```

Comment:
😂

Reply:
Yep. Was going to pre-comment this but you beat me to it :) 

❓ Should we add javadoc explaining if `IngestException` implies complete failure? Or partial failure?